### PR TITLE
Add hvdc and default shunt models

### PR DESCRIPTION
**Pull request by Lisrte**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: dynawaltz/src/test/java/com/powsybl/dynawaltz/xml/AbstractDynamicModelXmlTest.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import java.util.Objects;
+
+import static com.powsybl.commons.test.ComparisonUtils.compareTxt;
+
+/**
+ * @author Laurent Issertial <laurent.issertial at rte-france.com>
+ */
+public abstract class AbstractDynamicModelXmlTest extends AbstractConverterTest {
+
+    protected static Network network;
```
Comment (flo-dup) [inline dynawaltz/src/test/java/com/powsybl/dynawaltz/xml/AbstractDynamicModelXmlTest.java:44]: It's rather dangerous to have it static, tests might be on parallel runs. And you modify this network in `HvdcModelXmlTest`, and probably in `addStaticModels` too?

File: dynawaltz/src/main/java/com/powsybl/dynawaltz/models/buses/StandardBus.java [hunk 2]
```diff
@@ -90,4 +90,13 @@ public String getSwitchOffSignalVarName() {
...
     }
+
+    @Override
+    public String getUImpinVarName() {
+        return "@NAME@_U";
+    }
+
+    public String getUpuImpinVarName() {
+        return "@NAME@_Upu";
+    }
```
Comment (flo-dup) [inline dynawaltz/src/main/java/com/powsybl/dynawaltz/models/buses/StandardBus.java:101]: These var names do not exist for this model, we should either throw an exception or return Optional.empty()

**Review by flo-dup — APPROVED** [review]

File: dynawaltz/src/main/java/com/powsybl/dynawaltz/models/generators/AbstractGeneratorModel.java [hunk 3]
```diff
@@ -97,4 +97,39 @@ public String getSwitchOffSignalAutomatonVarName() {
...
     }
+
+    @Override
+    public String getQStatorPuVarName() {
+        return "generator_QStatorPu";
+    }
+
+    @Override
+    public String getLevelKVarName() {
+        return "reactivePowerControlLoop_levelK";
```
Comment (gautierbureau) [inline dynawaltz/src/main/java/com/powsybl/dynawaltz/models/generators/AbstractGeneratorModel.java:108]: I think we need to add a new child class for generators with RPCL

Comment (Lisrte) [reply dynawaltz/src/main/java/com/powsybl/dynawaltz/models/generators/AbstractGeneratorModel.java:108]: Will be fixed by #192

File: dynawaltz/src/main/java/com/powsybl/dynawaltz/models/buses/StandardBus.java
Comment (flo-dup) [inline dynawaltz/src/main/java/com/powsybl/dynawaltz/models/buses/StandardBus.java:1]: Now that the DefaultShuntModel has been added, I just realize, shouldn't we protect the StandardBus from being connected to any default model?
We should add `checkLinkedDynamicModels(bus.getShuntCompensatorStream(), "shunt", context);` to the checks, or do it on all type of connectables with something like `bus.getConnectedTerminalStream().map(Terminal::getConnectable).forEach(c -> checkLinkedDynamicModels(c, context));`

**Review by flo-dup — APPROVED** [review]

**Pull request merged**